Interni/NormeProgetto/components/norme-c3.tex

```diff
@@ -0,0 +1,475 @@
+		\item \textbf{interni}: riguardanti le dinamiche del gruppo;
+		\item \textbf{esterni}: di interesse ai committenti e al proponente, vengono consegnati nell'ultima versione approvata.
+	\end{itemize} 
+In particolare, i documenti formali prodotti saranno:
+\newpage
+{
+	\setlength{\freewidth}{\dimexpr\textwidth-1\tabcolsep}
+	\renewcommand{\arraystretch}{1.5}
+	\setlength{\aboverulesep}{0pt}
+	\setlength{\belowrulesep}{0pt}
+	\rowcolors{2}{AzzurroGruppo!10}{white}
+	\begin{longtable}{L{.210\freewidth} L{.6\freewidth} L{.080\freewidth}}
+		\toprule 
+		\rowcolor{AzzurroGruppo!30}
+		\textbf{Nome} & \textbf{Descrizione} & \textbf{Uso}\\
+		\toprule
+		\endhead		
+		\NdP{} & Contiene tutte le regole stabilite dai membri alle quali attenersi durante l'intera durata del progetto. & Interno \\ 
+		\PdP{} & Contiene la pianificazione di tutte le attività previste, comprendente il preventivo delle spese e una previsione dell'impegno in ore per ogni membro del gruppo. & Esterno \\
+		\PdQ{} & Descrive i criteri di valutazione della qualità impiegate dal gruppo & Esterno \\ 
+		\SdF{} & Contiene l'analisi dei capitolati messi a disposizione, evidenziandone pregi e difetti. Contiene, inoltre, il capitolato scelto dal gruppo & Interno \\
+		\AdR{} & Descrive i requisiti che il prodotto dovrà possedere per essere in linea con le richieste dei committenti. & Esterno \\ 	
+		\G{} & Elenco di tutti i termini presenti nella documentazione che, secondo i membri, necessitano una definizione al fine di chiarirne il significato e rimuovere eventuali ambivalenze. & Esterno \\  			
+		\bottomrule
+		\hiderowcolors
+		\caption{Nome, Descrizione ed uso dei documenti formali prodotti}
+	\end{longtable}
+}
+
+\item \textbf{Informali}: sono documenti non soggetti a versionamento o che non sono ancora stati approvati dal \textit{Responsabile di Progetto}. \newline 
+In particolare, i documenti informali prodotti saranno:
+{
+	\setlength{\freewidth}{\dimexpr\textwidth-1\tabcolsep}
+	\renewcommand{\arraystretch}{1.5}
+	\setlength{\aboverulesep}{0pt}
+	\setlength{\belowrulesep}{0pt}
+	\rowcolors{2}{AzzurroGruppo!10}{white}
+	\begin{longtable}{L{.210\freewidth} L{.6\freewidth} L{.080\freewidth}}
+		\toprule 
+		\rowcolor{AzzurroGruppo!30}
+		\textbf{Nome} & \textbf{Descrizione} & \textbf{Uso}\\
+		\toprule
+		\endhead		
+		Verbali interni & Contiene le informazioni e le decisioni prese durante gli incontri tra i membri del gruppo. & Interno \\ 
+		Verbali esterni & Comprende le informazioni ed i chiarimenti ricevuti durante gli incontri tra i membri ed il committente. & Esterno \\
+		\bottomrule
+		\hiderowcolors
+		\caption{Nome, Descrizione ed uso dei documenti informali prodotti}
+	\end{longtable}
+}
+\end{itemize}
+\subsubsection{Directory di un documento}
+Le directory prendono il nome dal documento contenuto, nella forma \textbf{NomeDelDocumento}.  Questa viene poi collocata,  a seconda del tipo di contenuto,  in una delle directory \textbf{Interni} o \textbf{Esterni}. 
```

Comment:
rimuovere doppio spazio prima di "Questa viene poi"

rimuovere doppio spazio prima di "a seconda del"

rimuovere doppio spazio prima di "in una delle"